libs/data-mapper/src/lib/utils/__test__/DataMapUtils.spec.ts

```diff
@@ -163,6 +164,18 @@ describe('utils/DataMap', () => {
       expect(resultEntries[3][1]).toBeTruthy();
     });
 
+    it('creates a simple conditional connection', () => {
+      simpleMap['ns0:Root'] = {
+        DataTranslation: {
+          EmployeeName: {
+            RegularFullTime: '$if(is-greater-than(/ns0:Root/ConditionalMapping/ItemPrice, /ns0:Root/ConditionalMapping/ItemQuantity))',
```

Comment:
I'm not sure this is a valid mapping? I could be wrong, but I thought we needed to use "if-then-else" in something like `if-then-else(is-greater-than(/ns0:Root/ConditionalMapping/ItemPrice, /ns0:Root/ConditionalMapping/ItemQuantity), $true, $nil`)

Reply:
I can send it to Sanjiv's team to confirm, but that is what we generate. I know that we also need to check if-then-else, but I was thinking including this simple test could be helpful for me as a sanity check.

Reply:
Let's chat offline about this